elastic/kibana
# [Inventory][Hosts] Make alert count consistent with the alerts page using a filter

Closes #202979 

## Summary

This PR fixes inconsistencies between the alerts shown on hosts view and inventory and the alerts page. The fix includes using consistent rule types and filters for both getting the alert count and filters applied on the alert page 

## How to Test

Service Alert ( Inventory )

https://github.com/user-attachments/assets/f3b626da-1a49-42dc-a989-48b13d15ae2c

Hosts view (+filters) Alerts (Inventory & Custom Threshold rule)


https://github.com/user-attachments/assets/2a490ad4-e2a4-43b5-b00f-d00ac27f9fd3

Single Host

https://github.com/user-attachments/assets/7c6a8cf7-f2a2-41f0-9f98-7e7543d4e7d5

cc: @roshan-elastic Ping to check the videos of the fix :)
